kubetest/anywhere.go

```diff
@@ -193,6 +194,12 @@ func newKubernetesAnywhere(project, zone string) (deployer, error) {
 		return nil, err
 	}
 
+	// Set NUM_NODES based on the kubernetes-anywhere-num-nodes flag.
+	// This env variable is then read by hack/ginkgo-e2e.sh.
```

Comment:
I'm pretty sure needing to set this is going to be an issue for sonobuoy. I'm fine to fix this here to unblock, but I think we're dodging the e2e framework really needing to compute this itself. cc @timothysc

Reply:
commented here: https://github.com/kubernetes/kubernetes/issues/74474#issuecomment-466825601